templates/etc/puppetserver/conf.d/webserver.conf.epp

```diff
@@ -24,13 +36,35 @@ webserver: {
     client-auth: want
     ssl-crl-path: <%= $pupmod::ssldir %>/crl.pem
```

Comment:
Why change the ssldir variable for the machine certs but not the ca and crl?

Reply:
@jeannegreulich They happen to live in different places on the server. It's a bit weird and probably needs to be fully refactored in the future to see if the puppetserver app has pinned down some of the legacy quirks.

Reply:
@jeannegreulich Good question. We probably need to fully refactor the module. This patch is only trying to fix the identified bug with as little potential damage as possible.